src/systems/physics/Physics.cc

```diff
@@ -166,9 +167,10 @@ class gz::sim::systems::PhysicsPrivate
           gz::physics::LinkFrameSemantics,
           gz::physics::ForwardStep,
           gz::physics::RemoveModelFromWorld,
-          gz::physics::sdf::ConstructSdfLink,
           gz::physics::sdf::ConstructSdfModel,
-          gz::physics::sdf::ConstructSdfWorld
+          gz::physics::sdf::ConstructSdfWorld,
+          gz::physics::GetLinkFromModel,
+          gz::physics::GetShapeFromLink
```

Comment:
Note that modifying `MinimumFeatureList` isn't backwards compatible, as documented above. So I'm adding the `Breaking change` label.